client/dashboard/app/dev-tools/preferences.scss

```
@@ -1,15 +1,29 @@
 @import '@wordpress/base-styles/variables';
 
-.preferences-helper__current-preferences {
+.preferences-helper__preference {
 	font-size: $font-size-small;
 	text-transform: none;
+	font-weight: normal;
 }
 
-.environment.is-prefs:not(:hover) .preferences-helper__current-preferences {
+.preferences-helper__preference-content {
+	padding-inline-start: 36px;
```

This will be 32px, so not quite the same. But maybe better to stick to standard spacing?

```suggestion
	padding-inline-start: $grid-unit-40;
```

---

Good call out! Updated.